### PR TITLE
C4F03 - Feature, record suggested synonym for a given word RO

### PR DESCRIPTION
**Pull request by omar-mohyeldin**
Issue #83 
@karimngr 
@AmrARaouf 
@Mirna-Yacout 
@nohahesham 
Please review

### CODE REVIEW

**Comment by omar-mohyeldin** [comment]
@m0hamed @NourhanMohamed does it look fine for you?

**Comment by AmrARaouf** [comment]
Omar please attach an issue number

**Comment by m0hamed** [comment]
it looks fine

**Comment by m0hamed** [comment]
actually the synonym should be either formal or slang. There is a migration in master that adds that column. please amend your method

**Comment by Mirna-Yacout** [comment]
- title
- the arrow calling the function is not pointing to the correct place
- else should be written in square brackets [else]

**Comment by omar-mohyeldin** [comment]
@m0hamed I can't find this info on master

**Comment by omar-mohyeldin** [comment]
@Mirna-Yacout what do you mean by title? we never did use titles in our UMLs. So what do you mean by them and what is its content? And are we obliged to add them even though we never did in our previous UMLs?

**Comment by m0hamed** [comment]
@Omar-Hossam git hub was misbehaving. it should be on master now

File: arability/app/models/synonym.rb [hunk 1]
```diff
@@ -128,56 +120,35 @@ def retrieve_synonyms(keyword)
...
-  #   database and returns a boolean result and the new synonym model
-  #   indicatiing success or failure of saving, and the synonym model to look
-  #   for reason of saving failure.
+  #   Feature, record suggested synonym for a given word. The function saves
+  #   synonym to database and returns boolean only or boolean and the new
+  #   synonym depending on the input parameters.
   # Parameters:
   #   synonym_name: string input parameter that represents the synonym name.
   #   keyword_id: integer input parameter representing the keyword id
   #     the synonym points to.
```
Comment (AmrARaouf) [inline arability/app/models/synonym.rb]: please align this line with the one above it. The body of the documentation is only indented by 3 spaces not 6 even if it is a continuation of another line

**Comment by Mirna-Yacout** [comment]
by title I mean a title for your story to indicate which story u are implementing, as far as I recently knew (that's why I did not require it before), it's necassary

**Comment by omar-mohyeldin** [comment]
@Mirna-Yacout @AmrARaouf please re-review

**Comment by omar-mohyeldin** [comment]
@nohahesham @karimngr please review

**Comment by Mirna-Yacout** [comment]
verified

**Comment by nohahesham** [comment]
verified

**Comment by ghost** [comment]
verified

**Pull request merged**